### PR TITLE
docs: bento client

### PR DESCRIPTION
**Pull request by sauyon**

### CODE REVIEW

File: docs/source/guides/client.rst [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        return await iris_clf_runner.predict.async_run(input_series)
+
+The client can be used in four ways:
+
+.. code-block:: python
+
+    res = client.classify(np.array([[4.9, 3.0, 1.4, 0.2]]))
+    res = await client.async_classify(np.array([[4.9, 3.0, 1.4, 0.2]]))
+    res = client.call("classify", np.array([[4.9, 3.0, 1.4, 0.2]]))
+    res = await client.async_call("classify", np.array([[4.9, 3.0, 1.4, 0.2]]))
```
Comment (aarnphm) [inline docs/source/guides/client.rst:37]: I think await can only be called in a async function. It is probably more helpful to provide the snippet code to how one can copy and run it.

**Review by aarnphm — APPROVED** [review]
Looks good to me.

File: docs/source/guides/client.rst [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+.. code-block:: python
+    @svc.api(input=Multipart(a=Text(), b=Text()), output=JSON())
+    def combine(a, b) -> dict[typing.Any, typing.Any]:
+        return {a: b}
+
+The client call would look like:
+
+.. code-block:: python
+    res = client.combine(a="a", b="b")
```
Comment (ssheng) [inline docs/source/guides/client.rst:64]:
```suggestion
.. code-block:: python

    res = client.combine(a="a", b="b")
```

File: docs/source/guides/client.rst [hunk 3]
```diff
@@ -0,0 +1,64 @@
+========================
+Using the BentoML Client
```
Comment (ssheng) [inline docs/source/guides/client.rst:2]:
```suggestion
Bento Client
```

**Review by ssheng — APPROVED** [review]

**Pull request merged**